api/views.py

```diff
@@ -325,6 +326,19 @@ def partners(request):
     return JsonResponse(data, safe=False)
 
 
+def create_magic_link(request):
```

Comment:
It is not specified how long a magic link should last. I think default is 5 minutes, which might be too little.